examples/configmap_reflector.rs

```diff
@@ -0,0 +1,42 @@
+#[macro_use] extern crate log;
+use std::collections::BTreeMap;
+
+use kube::{
+    api::{Api, Reflector},
+    client::APIClient,
+    config,
+};
+
+/// Example way to read secrets
+fn main() -> Result<(), failure::Error> {
+    std::env::set_var("RUST_LOG", "info,kube=trace");
+    env_logger::init();
+    let config = config::load_kube_config().expect("failed to load kubeconfig");
+    let client = APIClient::new(config);
+    let namespace = std::env::var("NAMESPACE").unwrap_or("default".into());
+
+    let resource = Api::v1ConfigMap(client).within(&namespace);
+    let rf = Reflector::new(resource).init()?;
+
+    // Can read initial state now:
+    rf.read()?.into_iter().for_each(|(name, d)| {
+        info!("try to read configmap");
+        let mut res = BTreeMap::new();
+        for (k, v) in d.data {
+            res.insert(k, v);
+        }
```

Comment:
this looks like a leftover copy of the data thing in secretmap, going to remove it

Reply:
yes, this is copied from secret reflector example